### PR TITLE
Add-EAP-action-6134, Change-trigger-msg-6380, Change-title-to-bold-9117

### PR DESCRIPTION
**Pull request by SwatiTh**

### CODE REVIEW

**Comment by SwatiTh** [comment]
@jannisvisser @gulfaraz - seems like the CI checks while running npm test are failing due to missing SECRETS(MC_API), this is failing in master as well.
Please suggest the workaround to fix this.

**Comment by jannisvisser** [comment]
@SwatiTh I believe we ran into this previously already as well. You need to add the same secrets as here (https://github.com/rodekruis/IBF-system/settings/secrets/actions) also in your own fork. 

Or even simpler: stop using your own fork, and simply push to branches on the upstream repository.

File: interfaces/IBF-dashboard/src/app/components/aggregates/aggregates.component.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
   .aggregate-header-item-label {
     margin: 8px 0;
+    font-weight: 600;
```
Comment (jannisvisser) [inline interfaces/IBF-dashboard/src/app/components/aggregates/aggregates.component.scss:11]: This is for the bold text, but that one is already reviewed according to DevOps, and I thought I saw it in a different PR already before?

**Review by jannisvisser — CHANGES_REQUESTED** [review]
@SwatiTh see comments already here in this PR, even though I am counting on a new PR from upstream feature branch to upstream master.

**Pull request merged**